### PR TITLE
Update .editorconfig

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -5,10 +5,10 @@ end_of_line = lf
 insert_final_newline = true
 charset = utf-8
 
-[*.{js,html}]
+[*.{js,json,html}]
 indent_style = space
 indent_size = 2
 
-[*.{json,scss}]
+[*.{scss,less}]
```
Comment (aju) [inline .editorconfig]: Should I merge those 2 blocks? @mr-mig @kdzwinel @matzimowski @vergilius  ??

**Comment by vergilius** [comment]
not sure about having these two options duplicated :rabbit2:

File: .editorconfig (hunk 1 above)
Comment (mr-mig) [inline .editorconfig]: yep :+1:

**Comment by mr-mig** [comment]
:+1:

**Comment by vergilius** [comment]
:+1:

**Pull request merged**